code/modules/projectiles/projectile/beams.dm

```diff
@@ -14,6 +14,8 @@
 	tracer_type = /obj/effect/projectile/laser/tracer
 	impact_type = /obj/effect/projectile/laser/impact
 
+	heat = 200
+
 /obj/item/projectile/beam/practice
```

Comment:
zero out the heat for practise beams. they're basically just harmless holograms

Reply:
It is already 0 if they deal 0 damage.